conda/__init__.py

```diff
@@ -41,7 +41,13 @@
 if os.getenv('CONDA_ROOT') is None:
     os.environ[str('CONDA_ROOT')] = sys.prefix
 
+#: The conda package directory.
 CONDA_PACKAGE_ROOT = abspath(dirname(__file__))
+#: The path within which to find the conda package.
+#:
+#: If `conda` is statically installed this is the site-packages. If `conda` is an editable install
+#: or otherwise uninstalled this is the git repo.
+CONDA_SOURCE_ROOT = dirname(CONDA_PACKAGE_ROOT)
```

Comment:
@jezdez @beeankha do either of you have experience defining docs for consts? Outside of defining them all at the top in the module docstring using the autodoc extension for sphinx seems to be the best option? Looks like urllib uses this.

Reply:
I don't have direct experience defining docs for consts, but after doing some poking around/research, it appears that the sphinx option is indeed best! 👍

Reply:
Yeah, autodoc is the way to go 👍🏻